src/main/java/ru/itmo/sd/deadliner2bot/model/Todo.java

```diff
@@ -26,6 +33,9 @@
     @Column(name = "completed")
     private boolean completed;
 
+    @Column(name = "daily_notifications_enabled")
+    private boolean dailyNotificationsEnabled;
```

Comment:
Should be setted to `false` by default

Reply:
There should be way more changes if you want to replicate constraints from Liquibase in the model.